website/static/js/institutionNodes.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var ko = require('knockout');
+var Raven = require('raven-js');
+
+var $osf = require('js/osfHelpers');
+
+
+var ViewModel = function(context) {
+    var self = this;
+    self.ctx = context;
+    self.allNodes = ko.observable();
```

Comment:
it doesn't need to be if I remember correctly, as I'm just assigning the whole list to it, rather than iterating and pushing to it.